scripts/dataset/index.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import generateRelease from './export/index.js';
+import logger from './logger/index.js';
+import publishRelease from './publish/index.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+export async function release({ publicationEnabled, removeLocalCopyEnabled, fileName }) {
+  const releaseDate = new Date();
+  const archiveName = fileName || `dataset-${releaseDate.toISOString().replace(/T.*Z/, '')}`;
```

Comment:
Again, difficult to read, so you remove everything after `T`? Keeping only the main year, month and day, right?

I would use dayjs, but else `/T.*$/` to be sure to remove everything